_docs/deprecated_and_archived_modules.md

```diff
@@ -14,4 +14,17 @@ purpose.  For example, a module that interacted with a discontinued 3rd party se
 If you come across an archived Vox Pupuli project that you are still using or think should not have been archived, [let us know][email].
 The decision to archive a module isn't necessarily final and can be undone.
 
+### Deprecation process
+
+* An issue needs to be raised on the module to discuss wether it should be archived or not. The decision will be made by lazy consensus (as described in our governance guidelines)
+* The README.md needs a bold message at the beginning
+  * It needs to mention that the module is archived
```

Comment:
It'd suggest to use must/should per [RFC 2119](https://tools.ietf.org/html/rfc2119)
```suggestion
  * It must mention that the module is archived
```